draft-ietf-tls-tls13.md

```diff
@@ -4376,7 +4376,7 @@ defined below:
 
     struct {
         uint16 length = Length;
-        opaque label<10..255> = "TLS 1.3, " + Label;
+        opaque label<10..255> = "tls13 " + Label;
```

Comment:
Maybe also change the max to 18? At minimum, sticking a short note somewhere on the reason for the label lengths would probably be worth doing. This affects interop, so this also needs a changelog entry.